storage-bigtable/src/lib.rs

```diff
@@ -1146,17 +1153,24 @@ impl LedgerStorage {
                     .await?;
             }
 
+            if entries_exist {
+                self.connection
+                    .delete_rows_with_retry("entries", &[slot_to_entries_key(slot)])
+                    .await?;
+            }
+
             self.connection
                 .delete_rows_with_retry("blocks", &[slot_to_blocks_key(slot)])
                 .await?;
         }
 
         info!(
-            "{}deleted ledger data for slot {}: {} transaction rows, {} address slot rows",
+            "{}deleted ledger data for slot {}: {} transaction rows, {} address slot rows, {} entry row",
             if dry_run { "[dry run] " } else { "" },
             slot,
             tx_deletion_rows.len(),
-            address_slot_rows.len()
+            address_slot_rows.len(),
+            if entries_exist { "with" } else {"WITHOUT"}
```

Comment:
Works for me as-is, I guess the all-caps is trying to draw attention to the fact that entries for this slot were not populated as opposed to just using `without`.

Reply:
Yes, that was my thought. I don't feel strongly about it, though, if you hate it 😅

Reply:
Ha nope - I'm fine with it hence the accompanying ship-it